src/BlockTypes/ProductStockIndicator.php

```diff
@@ -0,0 +1,46 @@
+<?php
+namespace Automattic\WooCommerce\Blocks\BlockTypes;
+
+/**
+ * ProductStockIndicator class.
+ */
+class ProductStockIndicator extends AbstractBlock {
+
+	/**
+	 * Block name.
+	 *
+	 * @var string
+	 */
+	protected $block_name = 'product-stock-indicator';
+
+	/**
+	 * API version name.
+	 *
+	 * @var string
+	 */
+	protected $api_version = '2';
+
+	/**
+	 * Get block supports. Shared with the frontend.
+	 * IMPORTANT: If you change anything here, make sure to update the JS file too.
+	 *
+	 * @return array
+	 */
+	protected function get_block_type_supports() {
+		return array(
+			'color'                  =>
+			array(
+				'link'  => false,
+				'background' => false,
+				'text' => true,
```

Comment:
Based on the PHP Coding Standard, we align `=` and `=>` under each other to increase the readability. You can see an example of that on https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#indentation. 

Don't worry, there's no need to manually do that. `npm run lint:php-fix` will do that job for you. 😉